joomla/joomla-cms
# [4.0] ES6 conversion of select-colour.js

Pull Request for Issue # .

### Summary of Changes
Script converted to ES6

### Testing Instructions
Edit an article and change the status: the select box should be either red, green or white depending on the state: published, unpublished, archived


### Expected result



### Actual result



### Documentation Changes Required
